v22.2/import-into.md

```diff
@@ -33,16 +33,17 @@ The user must have the `INSERT` and `DROP` [privileges](security-reference/autho
 
 #### Source privileges
 
-The source file URL does _not_ require the `ADMIN` role in the following scenarios:
+The user that runs `IMPORT INTO` to external storage does **not** require the `ADMIN` role in the following scenarios:
 
-- S3 and GS using `SPECIFIED` (and not `IMPLICIT`) credentials. Azure is always `SPECIFIED` by default.
+- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
 - [Userfile](use-userfile-for-bulk-operations.html)
 
-The source file URL _does_ require the `ADMIN` role in the following scenarios:
+{% include {{ page.version.version }}/misc/external-io-privilege.md %}
 
-- S3 or GS using `IMPLICIT` credentials
-- Use of a [custom endpoint](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/) on S3
-- [Nodelocal](cockroach-nodelocal-upload.html), [HTTP](use-a-local-file-server-for-bulk-operations.html), or [HTTPS](use-a-local-file-server-for-bulk-operations.html)
+The user that runs `IMPORT INTO` to external storage **does** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
```

Comment:
Same as the comment in destination-privileges.md I don't know if we want to give the user the impression that there are still operations that only admin can run, because thats not the case anymore. EXTERNALIOIMPLICITACCESS governs who can and cannot run against an ExternalStorage.